cypress/utils/index.ts

```diff
@@ -87,3 +87,15 @@ export const selectAntdOption = (dataCy: string, option: string) => {
     "not.be.visible"
   );
 };
+
+/**
+ * Given the label of a select input and the test of the option, this function will open the select and click on that option
+ * @param label
+ * @param optionText
+ * @returns
+ */
+export const selectDropdown = (label: string, optionText: string) => {
```

Comment:
nit. the name of this function is not that precise compared to the old function `selectAntdOption`. it could be nice to call it `selectLGOption` which would mirror the function in `test_utils`